templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+      </p>
+      <p>
+        Enterprise- grade long term support and maintenance ensure that you get reliable and fast turn-around time for bug fixes.
+      </p>
+      <p>
+        Check out the joint solutions offered by Canonical and our silicon partners.
+      </p>
+      <ul class="p-list--divided-bulleted ">
+        <li class="p-list__item">
+          Watch the on-demand panel to learn
+          <a href="https://ubuntu.com/engage/nvidia-gtc-2022">
+            How Telcos Leverage Canonical and NVIDIA for AI Infrastructure and
+            Accelerated Visualization
+          </a>
+        </li>
+        <li class="p-list__item">
+          Learn more about
+          <a
+            href="https://canonical.com/blog/openstack-yoga-on-ubuntu-lts-delivers-highly-performant-infrastructure-for-telcos-and-researchers-with-smartnics-and-dpus"
+          >
+            how NVIDIA and Canonical worked to bring OVN offstreaming to the DPU
+          </a>
+        </li>
+        <li class="p-list__item">
+          Find out how we work with Intel in the telecom industry today:
+          <a
+            href="https://ubuntu.com/blog/canonical-announces-ubuntu-22-04-lts-support-for-flexran-reference-software"
+          >
+            Ubuntu 22.04 LTS Support for FlexRAN Reference Software
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="u-sv2">
+      <hr />
+    </div>
+    <div class="col-6">
+      <h2 class="u-no-padding--top">Reduce development friction<br class="u-hide--medium u-hide--small" /> and time to market</h2>
+    </div>
+    <div class="col-6">
+      <div class="u-hide--medium u-hide--small u-sv1">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/66433671-build-your-own-stack-and-prototype-your-application.png",
+          alt="build your own stack and prototype your application illustration",
+          width="600",
+          height="336",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <p>
+        Ubuntu on workstations, servers, IOT devices, edge devices allows for seamless developer experience thus reducing cost and friction for application development and maintenance.
+      </p>
+      <p>
+        Pro-active pre GA work on next-generation software and silicon hardware through optimised Ubuntu offering ensures silicon features reach both internal and external users in the fastest way possible. Strong Ubuntu community engagement ensures constant evolution of hardware and software features.
+      </p>
+      <p>
+        Explore how the joint solutions offered by Canonical and our silicon partners accelerate the customers&rsaquo; project.
```

Comment:
```suggestion
        Explore how the joint solutions offered by Canonical and our silicon partners accelerate the customers' project.
```
You can just use a regular apostrophe here